mlir/test/Dialect/Linalg/fusion-elementwise-ops-no-remove-outs-deps.mlir

```diff
@@ -0,0 +1,18 @@
+// RUN: mlir-opt %s -p 'builtin.module(func.func(linalg-fuse-elementwise-ops{remove-outs-dependency=0}))' -split-input-file | FileCheck %s
+
+#identity = affine_map<(d0) -> (d0)>
+
+func.func @redudant_copy_with_target_burst_size_two(%arg: tensor<4xf32>) -> tensor<4xf32> attributes {plhw.toplevel} {
+  // CHECK-NOT: tensor.empty
+  %1 = linalg.generic {indexing_maps = [#identity, #identity], iterator_types = ["parallel"] } ins(%arg: tensor<4xf32>) outs(%arg: tensor<4xf32>) {
+    ^bb0(%in: f32, %out: f32):
+      %exp = arith.negf %in: f32
+      linalg.yield %exp : f32
+  } -> tensor<4xf32>
+  %2 = linalg.generic {indexing_maps = [#identity, #identity], iterator_types = ["parallel"] } ins(%1: tensor<4xf32>) outs(%arg: tensor<4xf32>) {
+    ^bb0(%in: f32, %out: f32):
+      %exp = arith.mulf %in,%in: f32
+      linalg.yield %exp : f32
+  } -> tensor<4xf32>
```

Comment:
I think you don't need the second generic to show the transformation doesn't happen? Maybe drop it?

Reply:
Correct, but since it is testing the fusion pass I thought I would keep some fusion going on for context. Otherwise, it is a bit weird. But sure I can make it.

Reply:
Fixed.